edx_proctoring/api.py

```diff
@@ -34,6 +71,11 @@ def get_exam_by_id(exam_id):
 
     Returns dictionary version of the Django ORM object
     """
+    proctored_exam = ProctoredExam.get_exam_by_id(exam_id)
+    if proctored_exam is None:
+        raise ProctoredExamNotFoundException
+
+    return proctored_exam.__dict__
```

Comment:
I see you are using this pattern often in this file. I think I noticed from @afzaledx's comment that this might be changing to returning output from the Serializer?

I think I would prefer output from the Serializer as well, but - either way - I think we should document what the schema looks like in the documentation comment string so that callers (in case they are calling into api.py as an in-proc library) know what to expect.